Fix `TripleDES` issue that's blocking `cryptography` dependency update

### Description

https://github.com/rucio/rucio/pull/7073

Errors:
```
E   assert not '/opt/venv/lib64/python3.9/site-packages/paramiko/pkey.py:100: CryptographyDeprecationWarning: TripleDES has been moved to cryptography.hazmat.decrepit.ciphers.algorithms.TripleDES and will be removed from this module in 48.0.0.\n  "cipher": algorithms.TripleDES,\n/opt/venv/lib64/python3.9/site-packages/paramiko/transport.py:259: CryptographyDeprecationWarning: TripleDES has been moved to cryptography.hazmat.decrepit.ciphers.algorithms.TripleDES and will be removed from this module in 48.0.0.\n  "class": algorithms.TripleDES,\n'
```

### Steps to reproduce

N/A

### Rucio Version

_No response_

### Additional Information

_No response_

--- a/requirements/requirements.dev.txt
+++ b/requirements/requirements.dev.txt
@@ -92,7 +92,7 @@ click==8.1.7
     #   pydoc-markdown
 coverage[toml]==7.6.1
     # via pytest-cov
-cryptography==42.0.8
+cryptography==43.0.1
     # via
     #   -r requirements.server.txt
     #   globus-sdk
@@ -177,7 +177,9 @@ globus-sdk==3.41.0
 google-auth==2.32.0
     # via -r requirements.server.txt
 greenlet==3.0.3
-    # via -r requirements.server.txt
+    # via
+    #   -r requirements.server.txt
+    #   sqlalchemy
 gssapi==1.8.3
     # via
     #   -r requirements.server.txt
@@ -278,7 +280,7 @@ packaging==24.1
     #   build
     #   pytest
     #   qbittorrent-api
-paramiko==3.4.0
+paramiko==3.4.1
     # via -r requirements.server.txt
 pathspec==0.12.1
     # via black

--- a/requirements/requirements.server.in
+++ b/requirements/requirements.server.in
@@ -21,7 +21,7 @@ boto3==1.34.142                                             # S3 boto protocol (
 xmlsec==1.3.13, !=1.3.14                                    # Required to install pyproject.toml-based projects; 1.3.14 excluded due to https://github.com/xmlsec/python-xmlsec/issues/314
 
 # All dependencies needed in extras for rucio server/daemons should be defined here
-paramiko==3.4.0                                             # ssh_extras; SSH2 protocol library (also needed in the server)
+paramiko==3.4.1                                             # ssh_extras; SSH2 protocol library (also needed in the server)
 kerberos==1.3.1                                             # kerberos_extras for client and server
 pykerberos==1.2.4                                           # kerberos_extras for client and server
 requests-kerberos==0.15.0                                   # kerberos_extras for client and server

--- a/requirements/requirements.server.txt
+++ b/requirements/requirements.server.txt
@@ -47,7 +47,7 @@ charset-normalizer==3.3.2
     # via requests
 click==8.1.7
     # via flask
-cryptography==42.0.8
+cryptography==43.0.1
     # via
     #   globus-sdk
     #   oic
@@ -136,7 +136,7 @@ oic==1.7.0
     # via -r requirements.server.in
 packaging==24.1
     # via qbittorrent-api
-paramiko==3.4.0
+paramiko==3.4.1
     # via -r requirements.server.in
 pbr==6.0.0
     # via stevedore

